sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,16 +1,13 @@
 ## Release History
 
-### 4.5.2b6 (Unreleased)
+### 4.6.0 (2024-03-14)
 
 #### Features Added
-
-#### Breaking Changes
+* GA release of hierarchical partitioning, index metrics and transactional batch.
 
 #### Bugs Fixed
 * Keyword arguments were not being passed down for `create_container_if_not_exists()` methods. See [PR 34286](https://github.com/Azure/azure-sdk-for-python/pull/34286).
 
-#### Other Changes
```

Comment:
I would add a note under `Other Changes` about the updates to type hinting - as it does involve the correction of some earlier type hints which people may notice.

Reply:
that's very fair, thanks for the heads up!